spec/models/employer_spec.rb

```diff
@@ -1,17 +1,27 @@
 require "rails_helper"
 
 describe Employer do
-  let(:name) { "My Company" }
-  let(:account_number) { 1234567 }
-  let(:employer) { FactoryBot.build_stubbed(:employer) }
-
   describe "initialisation" do
```

Comment:
Typically I wouldn't test the initialisation of a Rails model (unless there is something different or interesting happening).

By default Rails models [assign attributes to the instance](https://github.com/rails/rails/blob/e240daa6cc013b5a24fb692a8b82ce18dc44ad11/activemodel/lib/active_model/api.rb#L70-L84) when you pass them.